Deploy/Build-Module.ps1

```diff
@@ -43,7 +43,10 @@ New-ModuleManifest -Path:($FilePath_psd1) `
     -Description:('Powershell Module to run JumpCloud Active Directory Migration Utility.')
 
 # EndRegion Building New-JCModuleManifest
-
+# update psd1 with utf8 encoding
```

Comment:
Okay so I found that when you run New-ModuleManifest it encodes the .psd1 file as UTF16 which is fine but git sees files of this type as binary files so you never got a good git diff when you compared this file & grep can't read files encoded with UTF16 so I'm changing it back to UTF8 always when we generate the module manifest here